File: src/libs/ThirdParty.VBSuppliers/Properties/ExpediaRapid/SerializableClasses/Search/SearchResponseRoom.vb
Imports Newtonsoft.Json
Namespace ExpediaRapid.SerializableClasses.Search
    Public Class SearchResponseRoom

        <JsonProperty("id")>
        Public Property RoomID As String

        <JsonProperty("room_name")>
        Public Property RoomName As String

        <JsonProperty("rates")>
        Public Property Rates As New List(Of RoomRate)

    End Class

End Namespace

File: src/libs/iVectorOne.Suppliers/ExpediaRapid/SerializableClasses/Search/SearchResponseRoom.cs
namespace iVectorOne.CSSuppliers.ExpediaRapid.SerializableClasses.Search
{
    using System.Collections.Generic;
    using Newtonsoft.Json;

    public class SearchResponseRoom
    {

        [JsonProperty("id")]
        public string RoomID { get; set; }

        [JsonProperty("room_name")]
        public string RoomName { get; set; }

        [JsonProperty("rates")]
        public List<RoomRate> Rates { get; set; } = new List<RoomRate>();

    }

}